shared/bazel/rules/robotpy/wrapper.py

```diff
@@ -0,0 +1,33 @@
+import importlib
+import os
+import sys
+
+"""
+This file will wrap various semiwrap.tools executables. In the event that it fails
```

Comment:
This seems like it should be integrated with the tools themselves so that all semiwrap users can benefit from it?

Reply:
I'd be fine moving into semiwrap. I found it helpful to have a more readable (for me at least) output when I was bringing this up and figuring things out.

Reply:
This has since been updated to also modify the `PKGCFGS` envvar to get around not having to fully build and "install" a low level wheel to get higher level libraries building in parallel, so I think this is less of a candidate for getting moved into semiwrap now.